### ISSUE TITLE
None of the Glider events fire.

### ISSUE THREAD

**Issue by srameshr** [hipstersmoothie/react-glider#61]
```
 const gliderRef = useRef(null);
<Glider ref={gliderRef} onSlideVisible={(c,e) => console.log(c,e)} onLoad={(c,e) => console.log(c,e)}>
 {children}
</Glider>
```

Nothing gets logged to the console.

**Comment by kevinfarrugia** [comment]
I have opened a PR (#65) that resolves this issue.

Note that while debugging this, I identified and filed another issue #66.

**Comment by srameshr** [comment]
Is the maintainer active and accepting PR's?

**Comment by kevinfarrugia** [comment]
Yes I think so.

**Comment by hipstersmoothie** [comment]
:rocket: Issue was released in [`v2.2.0`](https://github.com/hipstersmoothie/react-glider/releases/tag/v2.2.0) :rocket:
